packages/styling/components/button.scss

```diff
@@ -1,8 +1,70 @@
-@mixin state-styling($background-color: inherit, $border-color: inherit, $color: inherit) {
-  background-color: $background-color;
-  border-color: $border-color;
-  color: $color;
-}
+@import "../variables";
+
+$btn-default-color: $grasgroen;
+$btn-default-bg: $wit;
+$btn-default-border: $grasgroen;
+
+$btn-primary-color: $wit;
+$btn-primary-bg: $grasgroen;
+$btn-primary-border: $grasgroen;
+
+$btn-link-disabled-color: lighten(#000, 46.7%);
+
+$dso-button-border-radius: $border-radius-base;
+$dso-button-border-width: 1px;
+
+$dso-button-size: $u7;
```

Comment:
Deze wordt alleen voor een `min-width` gebruikt, zullen we 'm specifieker benoemen?

Reply:
Ik stel voor om hem aan te passen in `$dso-button-min-width`